scripts/spoolman.sh

```diff
@@ -0,0 +1,281 @@
+#!/usr/bin/env bash
+
+#=======================================================================#
+# Copyright (C) 2020 - 2024 Dominik Willner <th33xitus@gmail.com>       #
+#                                                                       #
+# This file is part of KIAUH - Klipper Installation And Update Helper   #
+# https://github.com/dw-0/kiauh                                         #
+#                                                                       #
+# This file may be distributed under the terms of the GNU GPLv3 license #
+#=======================================================================#
+
+# Error Handling
+set -e
+
+function install_spoolman() {
+
+  pushd "${HOME}" &> /dev/null || exit 1
+
+  dependency_check curl jq
+
+  if [[ ! -d "${SPOOLMAN_DIR}" && -z "$(ls -A "${SPOOLMAN_DIR}" 2> /dev/null)" ]]; then
+    status_msg "Downloading spoolman..."
+    setup_spoolman_folder
+    status_msg "Downloading complete"
+    start_install_script
+    advanced_config_prompt
+  else
+    ### In case spoolman is "incomplete" rerun install script
+    if get_spoolman_status | grep -q "Incomplete!"; then
+      start_install_script
+      exit 1
+    fi
+
+    ok_msg "Spoolman already installed"
+    exit 1
+  fi
+
+  enable_moonraker_integration_prompt
+  patch_spoolman_update_manager
+
+  do_action_service "restart" "moonraker"
+}
+
+function update_spoolman() {
+  ### stop and disable old spoolman service
+  do_action_service "stop" "Spoolman"
+  do_action_service "disable" "Spoolman"
+
+  mv "${SPOOLMAN_DIR}" "${SPOOLMAN_DIR}_old"
+
+  setup_spoolman_folder
+  cp "${SPOOLMAN_DIR}_old/.env" "${SPOOLMAN_DIR}/.env"
+
+  start_install_script
+
+  rm -rf "${SPOOLMAN_DIR}_old"
+}
+
+function remove_spoolman(){
+  if [[ -d "${SPOOLMAN_DIR}" ]]; then
+    status_msg "Removing spoolman service..."
+    do_action_service "stop" "Spoolman"
+    do_action_service "disable" "Spoolman"
+    sudo rm -f "${SYSTEMD}/Spoolman.service"
+    ok_msg "Spoolman service removed!"
+
+    status_msg "Removing spoolman directory..."
+    rm -rf "${SPOOLMAN_DIR}"
+    ok_msg "Spoolman directory removed!"
+  fi
+
+  if [[ -d "${SPOOLMAN_DB_DIR}" ]]; then
+    status_msg "Removing spoolman db..."
+    rm -rf "${SPOOLMAN_DB_DIR}"
+    ok_msg "Spoolman db removed!"
+  fi
+
+  print_confirm "Spoolman successfully removed!"
+}
+
+function update_moonraker_configs() {
+  local moonraker_configs regex
+  regex="${HOME//\//\\/}\/([A-Za-z0-9_]+)\/config\/moonraker\.conf"
+  moonraker_configs=$(find "${HOME}" -maxdepth 3 -type f -regextype posix-extended -regex "${regex}" | sort)
+
+  for conf in ${moonraker_configs}; do
+    if ! grep -Eq "^\[update_manager KlipperScreen\]\s*$" "${conf}"; then
```

Comment:
```suggestion
    if ! grep -Eq "^\[update_manager Spoolman\]\s*$" "${conf}"; then
```
Copy paste mistake? :)